tasks/db_deploy/orca_sql.py

```diff
@@ -569,6 +570,8 @@ def granules_table_sql() -> TextClause:
             IS 'AWS step function execution id';
         COMMENT ON COLUMN granules.ingest_time
             IS 'Date and time the granule was originally ingested into ORCA.';
+        COMMENT ON COLUMN granules.cumulus_createdAt
+            IS 'Cumulus timestamp as part of the sync granule task output';
```

Comment:
Description should describe the data. I would maybe change to something like `Date/Time data was originally ingested into cumulus.` or other descriptive sentence.